internal/caching/index.ts

```
@@ -152,6 +152,11 @@ class CacheManagerExtends {
       WAL: true,
     });
   }
+  protected isPrimaryError(err: Error, callback?: Function) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)